### DISCUSSION TITLE
Synchronisation klappt nicht / Synchronization does not work

### DISCUSSION THREAD

**Discussion by HKaufmann2** [sboesen/remotely-sync#140] [Q&A]
I have installed Obsidian version 1.1.16 with your remotely-save version 0.5.25 on a Windows 7 computer and wanted to upload my Vault to my cloud from pCloud. 
Doesn't work, because my login data for the cloud is not requested. 
I got the code on the website https://remotelysave.com/user/profile. 
I only get the error message "Falied to connect". 
What can I do? Am I doing something wrong? 
Greetings Holger

Habe auf einen Windows 7 Rechner das Obsidian Version 1.1.16 mit Ihrem remotely-save Version 0.5.25 installiert und wollte meine Vault in meine Cloud von pCloud hochladen. 
Klappt nicht, da ja auch nicht meine Logindaten für die Cloud abgefragt wird. 
Habe den Code auf der Webseite https://remotelysave.com/user/profile geholt.
Bekomme nur die Fehlermeldung "Falied to connect".

Was kann man da machen? Mache ich etwas verkehrt?
Gruß Holger

**Comment by kadisonm** [comment]
Hello,

The provided links you have sent are not for our plugin but instead for [this one](https://github.com/remotely-save/remotely-save/tree/master). I recommend you open an issue there. If I am wrong please let me know.

Thanks

**Comment by HKaufmann2** [comment]
Hello Kadison,
Thank you for the response.
Greetings Holger
